### DISCUSSION TITLE
Hubitat support

### DISCUSSION THREAD

**Discussion by yozzozo** [toverainc/willow#190] [General]
Has anyone thought about Hubitat support for local device control, for those invested in that platform?

**Comment by kristiankielhofner** [comment]
I've looked at it before and from what I could tell based on available documentation for their "Maker API" integration with Willow isn't possible via that route. I'm vaguely aware they have apps or some kind of ecosystem to enable things like Willow but I haven't dug much further.

I'd like to support it but the docs, etc alone put it back on the shelf. For openHAB (as an example) I'd never even seen it before and we had it running with Willow in a couple hours. Based on everything I've seen with Hubitat the integration work on the Willow and Hubitat sides would be significant. But obviously I've never used it, seen it, etc and there's a decent chance I've just missed something.
